cartridge-cli.lua

```diff
@@ -4109,23 +4114,51 @@ local function stop_all(args)
     if processes == nil then return nil, err end
 
     local errors = {}
+    local warnings = {}
 
     -- stop instances
-    for _, process_args in pairs(processes) do
-        local instance_status_str = string.format('Stopping %s', process_args.instance_id)
+    for _, process in pairs(processes) do
+        local instance_status_str = string.format('Stopping %s', process.instance_id)
         local res_str
-
-        local ok, err = stop_process(process_args)
-        if not ok then
-            table.insert(errors, string.format('%s: %s', process_args.instance_id, err))
-            res_str = colored_msg('FAILED', ERROR_COLOR_CODE)
+        local err, warning
+
+        local status, status_err = process:get_status()
+
+        if status == nil then
```

Comment:
Looks I've seen similar piece of code here https://github.com/tarantool/cartridge-cli/pull/229/files#diff-65269a74ae0126ab3c284f41ebda16adR3771

Could you generalize it?

Reply:
There is an important difference between start and stop: `stop` interprets `Already stopped` as an warning, `start` interprets `Already started` as an error.

Moreover, I sure that generalizing starting and stopping will cause many errors. (for example, these command has different `already done` status)